site/role/manifests/tomcat_role.pp

```diff
@@ -0,0 +1,7 @@
+class role::tomcat_role {
+
+    include profile::base
```

Comment:
Hey we are using a different naming scheme, you can use profile::baseline::linux